src/Microsoft.DotNet.Wpf/src/PresentationFramework/System/Windows/Documents/TextStore.cs

```diff
@@ -2909,6 +2909,20 @@ private void PrepareAttributes(InputScope inputScope, double fontSize, FontFamil
             }
         }
 
+        private void ClearPrepareAttributes()
+        {
+            for (int i = 0; i < _preparedattributes.Count; i++)
+            {
+                var attributeVal = (UnsafeNativeMethods.TS_ATTRVAL)_preparedattributes[i];
+                if (attributeVal.val != null && attributeVal.val.vt == (short)NativeMethods.tagVT.VT_UNKNOWN && attributeVal.val.data1.Value != IntPtr.Zero)
+                {
+                    Marshal.Release(attributeVal.val.data1.Value);
```

Comment:
```suggestion
                    Marshal.Release(attributeVal.val.data1);
```

Reply:
@kant2002 Can I directly release the `data1`, not the `data1.Value`?

The type of the `data1` is `SecurityCriticalDataForSet<IntPtr>`

Reply:
https://github.com/dotnet/wpf/pull/7161/files#diff-e2eeb8d873cc40052e4a8c9960d7ee9c97be58b6e454f7f4e0d370e4aef05805L4371 

Oops, I was late. Yes, that's merely reaction to *latest* changes.